### PR TITLE
templates: allow no logo

### LINKED ISSUE #48: Allow `THEME_LOGO` to be `None`.
Currently, the module does not allow setting `THEME_LOGO = None`.

### PR DESCRIPTION
**Pull request by omelkonian**
* FIX Fixes issue when `THEME_LOGO` was set to `None` (closes #48).

### CODE REVIEW

File: invenio_theme/templates/invenio_theme/page_cover.html [hunk 1]
```diff
@@ -43,7 +43,9 @@
     <div class="row">
       <div class="col-md-6 col-md-offset-3">
         {%- block brand %}
-        <a href="/"><img class="text-center masthead-brand" src="{{ url_for('static', filename=config.THEME_LOGO)}}" alt="{{_(config.THEME_SITENAME)}}" width="250px"/></a>
+            {%- if config.THEME_LOGO %}
```
Comment (lnielsen) [inline invenio_theme/templates/invenio_theme/page_cover.html]: If not set it should likely print ``THEME_SITENAME`` instead? @egabancho

Comment (egabancho) [inline invenio_theme/templates/invenio_theme/page_cover.html]: I don't see why not

Comment (omelkonian) [inline invenio_theme/templates/invenio_theme/page_cover.html]: What if `SITENAME=None`?

Comment (jirikuncar) [inline invenio_theme/templates/invenio_theme/page_cover.html]: Then just leave it completely empty ...

**Comment by jirikuncar** [comment]
@omelkonian please use just 2 spaces for indentation in the template. It would be nice if you could also break the very long lines. Can you also add spaces inside double curly brackets for better readability (`<div>{{ _('Foo') }}</div>`). Thanks

**Pull request merged**